deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -61,28 +60,32 @@ def build_graph(self):
         """
     self.atom_features = Feature(shape=(None, self.n_atom_feat))
     self.pair_features = Feature(shape=(None, self.n_pair_feat))
-    combined = Combine_AP(in_layers=[self.atom_features, self.pair_features])
     self.pair_split = Feature(shape=(None,), dtype=tf.int32)
     self.atom_split = Feature(shape=(None,), dtype=tf.int32)
     self.atom_to_pair = Feature(shape=(None, 2), dtype=tf.int32)
-    weave_layer1 = WeaveLayer(
+    weave_layer1 = WeaveLayerFactory(
```

Comment:
Is this correct? `WeaveLayerFactory` should return `[weave_layer1, weave_layer2]` right?

Reply:
yes weave_layer1 here is a tuple of tensors

but it would be more accurate to call them 
```python
weave_layer1A, weavelayer1P
```